main/zoe/guide/contract-installation-and-instance.md

```diff
@@ -0,0 +1,98 @@
+# Contract installation and instance
+
+Zoe has two methods to "create a contract": 
```

Comment:
```suggestion
Zoe has two steps to "create a contract": 
```
Two methods is correct, but it sounds like the user has a choice between them, rather than needing to call both at some point in the process.